OpenRA.Game/ModData.cs

```diff
@@ -185,6 +185,9 @@ public void Dispose()
 
 	public interface ILoadScreen : IDisposable
 	{
+		/// <summary>Command line parameters that are not persisted.</summary>
+		LaunchArguments GetLaunchArguments();
```

Comment:
To address @phrohdoh's previous question and tidy the PR a bit, change this to `LaunchArguments Launch { get; }`

Reply:
I was missing that `get`.